src/tabookey-gasless/RelayClient.js

```diff
@@ -51,7 +51,17 @@ class RelayClient {
    *          Note that the resulting gasPrice must be accepted by relay (above its minGasPrice)
    *
    *manual settings: these can be used to override the default setting.
-   *    relayUrl, relayAddress - avoid lookup on relayHub for relays, and always use this URL/address
+   *    predefinedRelay - avoid lookup on relayHub for relays, and always use this predefined relay. 
+   *       An example of `predefinedRelay` Object :
+   *        {
+   *          RelayServerAddress: '0x73a652f54d5fd8273f17a28e206d47f5bd1bc06a',
+   *          MinGasPrice: 20000000000,
+   *          Ready: true,
```

Comment:
We should only mention the fields that are actually required (address, url and fee).

Reply:
You are right. Fixed.